### DISCUSSION TITLE
Slow performance using Helix

### DISCUSSION THREAD

**Discussion by kirawi** [wezterm/wezterm#2727] [Q&A]
I'm having poor performance on my A515-54-51DJ when trying to use Helix Editor. On the other hand, Alacritty is very fast. What can I do to debug this issue?

**Comment by yutkat** [comment]
I don't use Helix, but in general this problem is often improved by this config.

`wezterm.lua`
```lua
return {
  animation_fps = 1,
  cursor_blink_rate = 0,
  cursor_blink_ease_in = "Constant",
  cursor_blink_ease_out = "Constant",
}
```
